README.md

```diff
@@ -28,7 +28,7 @@ In general, contracts implement multiple types of methods:
 #### Read (views)
 Methods defined as views are free to use. They do not need to be part of a transaction and can simply be executed on a node. They can't perform any change to the state.
 #### Write
-Write methods need to be executed by a transaction and will cost fees to execute. Fees are refered as gas and the needed amount will depend on the written data's complexity. They cannot return any value but can throw.
+Write methods need to be executed by a transaction and will cost network fees to execute. Fees are refered as gas and the needed amount will depend on the written data's complexity. They cannot return any value but can throw.
```

Comment:
I'm nitpicking but from my side I think it's important to mention that fees are network fees.